multimodal/src/autogluon/multimodal/learners/default_learner.py

```diff
@@ -0,0 +1,2906 @@
+    MULTICLASS,
+    NER,
+    NER_RET,
+    NUMERICAL,
+    OBJECT_DETECTION,
+    OCR_TEXT_DETECTION,
+    OCR_TEXT_RECOGNITION,
+    OVERALL_F1,
+    RAY_TUNE_CHECKPOINT,
+    REGRESSION,
+    ROIS,
+    SCORE,
+    TEXT,
+    TEXT_NER,
+    UNIFORM_SOUP,
+    XYWH,
+    Y_PRED,
+    Y_PRED_PROB,
+    Y_TRUE,
+    ZERO_SHOT_IMAGE_CLASSIFICATION,
+)
+from ..data.datamodule import BaseDataModule
+from ..data.infer_types import (
+    infer_column_types,
+    infer_label_column_type_by_problem_type,
+    infer_problem_type_output_shape,
+    infer_rois_column_type,
+    is_image_column,
+)
+from ..data.preprocess_dataframe import MultiModalFeaturePreprocessor
+from ..matcher import MultiModalMatcher
+from ..models.utils import get_model_postprocess_fn
+from ..optimization.lit_distiller import DistillerLitModule
+from ..optimization.lit_mmdet import MMDetLitModule
+from ..optimization.lit_module import LitModule
+from ..optimization.lit_ner import NerLitModule
+from ..optimization.losses import RKDLoss
+from ..optimization.utils import (
+    get_loss_func,
+    get_metric,
+    get_norm_layer_param_names,
+    get_trainable_params_efficient_finetune,
+)
+from ..problem_types import PROBLEM_TYPES_REG, ProblemTypeProperty
+from ..utils import (
+    AutoMMModelCheckpoint,
+    AutoMMModelCheckpointIO,
+    CustomUnpickler,
+    DDPCacheWriter,
+    ExportMixin,
+    LogFilter,
+    apply_log_filter,
+    assign_feature_column_names,
+    average_checkpoints,
+    check_if_packages_installed,
+    compute_grad_steps,
+    compute_num_gpus,
+    compute_score,
+    convert_pred_to_xywh,
+    create_fusion_data_processors,
+    create_fusion_model,
+    data_to_df,
+    evaluate_coco,
+    extract_from_output,
+    filter_hyperparameters,
+    get_available_devices,
+    get_config,
+    get_detection_classes,
+    get_fit_complete_message,
+    get_fit_start_message,
+    get_local_pretrained_config_paths,
+    get_minmax_mode,
+    get_mixup,
+    get_stopping_threshold,
+    hyperparameter_tune,
+    infer_dtypes_by_model_names,
+    infer_metrics,
+    infer_precision,
+    infer_scarcity_mode_by_data_size,
+    init_df_preprocessor,
+    init_pretrained,
+    list_timm_models,
+    load_text_tokenizers,
+    logits_to_prob,
+    merge_bio_format,
+    modify_duplicate_model_names,
+    object_detection_data_to_df,
+    predict,
+    process_batch,
+    save_pretrained_model_configs,
+    save_result_df,
+    save_text_tokenizers,
+    select_model,
+    setup_detection_train_tuning_data,
+    setup_save_path,
+    split_hyperparameters,
+    tensor_to_ndarray,
+    turn_on_off_feature_column_info,
+    update_config_by_rules,
+    update_hyperparameters,
+    update_tabular_config_by_resources,
+    upgrade_config,
+)
+from ..utils.log import get_fit_complete_message, get_fit_start_message
+from .abstract_learner import AbstractMultiModalLearner
+
+logger = logging.getLogger(__name__)
+
+
+class DefaultLearner(AbstractMultiModalLearner):
```

Comment:
Consider unifying the name. Maybe `AbstractMultiModalLearner ` -> `AbstractLearner`.